_data/chains/eip155-4545.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Emoney Network Mainnet",
+  "chain": "Emoney",
+  "rpc": [
+    "https://rpc-publicnode.emoney.io/",
+    "https://public-node1-rpc.emoney.network/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Emoney Coin",
+    "decimals": 18,
+    "symbol": "EMYC"
+  },
+  "infoURL": "https://emoney.io/",
+  "shortName": "emoney",
+  "chainId": 4545,
+  "networkId": 4545,
+  "icon": "emoney",
+  "explorers": [
+    {
+      "name": "EMoney Explorer",
+      "url": "https://explore.emoney.network",
+      "icon": "emoney",
+      "standard": "EIP3091"
```

does not seem EIP3091 compatible
```suggestion
      "standard": "none"
```